pkg/printers/internalversion/describe.go

```diff
@@ -1535,6 +1535,8 @@ func describeDaemonSet(daemon *extensions.DaemonSet, events *api.EventList, runn
 		printAnnotationsMultiline(w, "Annotations", daemon.Annotations)
 		w.Write(LEVEL_0, "Desired Number of Nodes Scheduled: %d\n", daemon.Status.DesiredNumberScheduled)
 		w.Write(LEVEL_0, "Current Number of Nodes Scheduled: %d\n", daemon.Status.CurrentNumberScheduled)
+		w.Write(LEVEL_0, "Number of Nodes Scheduled with Up-to-date Pods: %d\n", daemon.Status.UpdatedNumberScheduled)
```

Comment:
Yeah, Number is redundant and generally I would prefer to keep these consistent with the D/RC/RS status but it got out of hand...